src/commands/nickname.ts

```diff
@@ -273,99 +261,97 @@ export default class NicknameCommand implements ApplicationCommand {
             ],
         });
 
-        const message = await command.fetchReply();
-        ongoingSuggestions[message.id] = {
-            nickNameUserId: user.id,
-            nickName: nickname,
-        };
+        const collector = reply.createMessageComponentCollector({
+            componentType: ComponentType.Button,
+            time: 1000 * 60 * 5,
```

Comment:
lager die magic number in ne variable aus

Reply:
d6154c9bfe050d989fe3632af20c618d5197c62a